src/rust/ensogl/lib/text/src/component/area.rs

```diff
@@ -949,45 +955,40 @@ impl application::View for Area {
-               Self::self_shortcut(shortcut::Action::release (&[],&[mouse::PrimaryButton])                                      , "stop_newest_selection_end_follow_mouse"),
-               Self::self_shortcut(shortcut::Action::press   (&[Key::Meta],&[mouse::PrimaryButton])                             , "add_cursor_at_mouse_position"),
-               Self::self_shortcut(shortcut::Action::press   (&[Key::Meta],&[mouse::PrimaryButton])                             , "start_newest_selection_end_follow_mouse"),
-               Self::self_shortcut(shortcut::Action::release (&[Key::Meta],&[mouse::PrimaryButton])                             , "stop_newest_selection_end_follow_mouse"),
-               Self::self_shortcut(shortcut::Action::press   (&[Key::Meta,Key::Character("a".into())],&[])                      , "select_all"),
-               Self::self_shortcut(shortcut::Action::press   (&[Key::Meta,Key::Character("c".into())],&[])                      , "copy"),
-               Self::self_shortcut(shortcut::Action::press   (&[Key::Meta,Key::Character("v".into())],&[])                      , "paste"),
-//               Self::self_shortcut(shortcut::Action::release (&[Key::Meta,Key::Character("z".into())],&[])                      , "undo"),
-//               Self::self_shortcut(shortcut::Action::release (&[Key::Meta,Key::Character("y".into())],&[])                      , "redo"),
-//               Self::self_shortcut(shortcut::Action::release (&[Key::Meta,Key::Shift,Key::Character("z".into())],&[])           , "redo"),
-//                Self::self_shortcut(shortcut::Action::press   (&[Key::Escape]                          , shortcut::Pattern::Any) , "undo"),
-        ]
+        use shortcut::ActionType::*;
+        (&[ (Press       , "left"                    , "cursor_move_left")
+          , (Press       , "right"                   , "cursor_move_right")
+          , (Press       , "up"                      , "cursor_move_up")
+          , (Press       , "down"                    , "cursor_move_down")
+          , (Press       , "meta left"               , "cursor_move_left_word")
+          , (Press       , "meta right"              , "cursor_move_right_word")
+          , (Press       , "meta alt left"           , "cursor_move_left_of_line")
+          , (Press       , "meta alt right"          , "cursor_move_right_of_line")
+          , (Press       , "shift left"              , "cursor_select_left")
+          , (Press       , "shift right"             , "cursor_select_right")
+          , (Press       , "meta shift left"         , "cursor_select_left_word")
+          , (Press       , "meta shift right"        , "cursor_select_right_word")
+          , (Press       , "shift up"                , "cursor_select_up")
+          , (Press       , "shift down"              , "cursor_select_down")
+          , (Press       , "backspace"               , "delete_left")
```

Comment:
```suggestion
          , (Press       , "backspace"               , "delete_left")
          , (Press       , "delete"                       , "delete_right")
```

Reply:
Please test if it works correctly as I dont have keyboard with delete button :D